scripts/superdesk-desks/index.js

```diff
@@ -13,6 +13,8 @@ import * as ctrl from './controllers';
 import * as directive from './directives';
 import { DesksFactory } from './services';
 
+console.assert(directive.DeskSelect, directive.DeskSelect);
```

Comment:
Is this forgotten or should we have it? I think Angular will fail anyway because of missing directives, so it's probably safe to remove.